third_party/silabs/SiWx917_sdk.gni

```diff
@@ -24,6 +24,14 @@ import("silabs_board.gni")
 
 examples_plat_dir = "${chip_root}/examples/platform/silabs/SiWx917"
 
+declare_args() {
+  # Enable the Alarm Based Wakeup for 917 SoC when sleep is enabled
+  alarm_based_wakeup = false
```

Comment:
maybe prefixing with `si91x_` would clarify what they are used for

Reply:
added `si91x_` prefix for these alarm based build options